src/styles/card.scss

```
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__img {
+    margin: auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review_point {
+    margin: 0;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $main-color;
```

Don't add unneeded empty lines
```suggestion
    margin: 0;
    font-size: 10px;
    font-weight: 400;
    line-height: 14px;
    color: $main-color;
```